### DISCUSSION TITLE
Should error messages passed through `strings` to `DatePicker` include a period?

### DISCUSSION THREAD

**Discussion by Rinzwind** [microsoft/fluentui#25670] [Q&A]
When no `strings` prop is passed to `DatePicker`, defaults are used in which the string for `isRequiredErrorMessage` does not include a period:

<img width="321" alt="1" src="https://user-images.githubusercontent.com/1611248/201896511-3b97d1b6-9aca-49a2-8c88-941af709eddc.png">

I’m not sure whether that is in line with the [Microsoft Style Guide](https://learn.microsoft.com/en-us/style-guide); compare ‘UI text or strings’ and ‘Error messages’ in the table on the [page ‘Formatting common text elements’](https://learn.microsoft.com/en-us/style-guide/text-formatting/formatting-common-text-elements) where the examples for the former do not include a period but the examples for the latter do:

<img width="661" alt="2" src="https://user-images.githubusercontent.com/1611248/201896576-9664cdaf-ac81-4908-81c5-f9099da9c8a2.png">

<img width="663" alt="3" src="https://user-images.githubusercontent.com/1611248/201896608-647797c1-8de8-41e5-9d95-a4b0a9985864.png">

See also the [page on ‘Periods’](https://learn.microsoft.com/nl-nl/style-guide/punctuation/periods).

Conversely, the default string did include a period at some point:

https://github.com/microsoft/fluentui/blob/fb8773239fa9476ab3dff28a483a1c758dac99ba/packages/date-time/src/components/DatePicker/defaults.ts#L6-L7

It was dropped in commit c838c580546dffb4:

https://github.com/microsoft/fluentui/blob/c838c580546dffb42b3eda95f55899c93c14b396/packages/date-time/src/components/DatePicker/defaults.ts#L11-L12

I can’t quite make out from the commit message what the motivation for that change was though. Could some further clarification be provided on the proper use of punctuation for these error messages?

**Comment by Rinzwind** [comment]
@ecraig12345 could I ask you to take a look at this question, as it relates to the change you did in commit c838c580546d in which the periods were dropped from the defaults for `isRequiredErrorMessage` and `invalidInputErrorMessage` in `DatePicker`? I’m wondering what in general the guideline is for the use of punctuation in these and similar messages (like `errorMessage` in `Dropdown`).

**Comment by ling1726** [comment]
@paulgildea  @gouttierre  would you know anything about how the microsoft style guide should be applied to default strings in Fluent UI?
